cfe_internal/enterprise/federation/federation.cf

```diff
@@ -0,0 +1,210 @@
+body file control { inputs => { "$(sys.libdir)/stdlib.cf" }; }
+
+bundle agent federation_config
+{
+  vars:
+    enterprise_edition.(policy_server|am_policy_hub)::
+      "federation_dir" string => "/opt/cfengine/federation";
+      "bin_dir" string => "$(federation_dir)/bin";
+      "path" string => "$(federation_dir)/cfapache/federation-config.json";
+  classes:
+    enterprise_edition.(policy_server|am_policy_hub)::
+      "federation_enabled"
```

Comment:
If we don't namespace this policy, then I think we should prefix namespace scoped classes with cfengine_internal

Reply:
ugh, long names.